### DISCUSSION TITLE
لینکهای اموزش نرم افزار

### DISCUSSION THREAD

**Discussion by itsmostafa75** [electron-v2ray/Free-V2ray-AutoRun-Advanced#10] [General]
سلام در قسمتی از اپ،لینک اموزش قرار دادین
 وقتی روی هر کدوم از لینکها میزنم باز نمیشه
باید چیکار کنم
میشه راهنمایی بفرمایید

**Comment by electron-v2ray** [comment]
موس ببرید روی لینک ها به سمت دکستاپ بکشید یا فایل index.zip روی صفحه اصلی گیت هاب دانلود کنید

**Comment by itsmostafa75** [comment]
سلام
ایا راهی هست تمام کانفیگهای موجود رو،بصورت یکجا استخراج کرد
یا مثلا اگر ۱۰ تا لینک رو انتخاب کردیم،تمام کانفیگهای اون ده لینک،یک جا داخل یک فایل باشه و بشه خارج از نرم افزار از اون فایل استفاده کرد؟

**Reply by electron-v2ray** [reply]
پوشه asset  سه فایل رو با نوت پد باز کنید
